server/src/bin/main.rs

```diff
@@ -34,9 +35,29 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
         })
         .transpose()?;
 
+    let remove_quote_callback = Arc::new(Mutex::new(|_quotes: Vec<Quote>| { /* Do nothing */ }));
```

Comment:
i didn't realize this is the server code and not the test code, yeah this needs to do something with the callback msg bus